### PR TITLE
Make sure all features are included in spring-security-support.adoc.

### PR DESCRIPTION
**Pull request by chenrujun**
As title.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @saragluna , @yiliuTo .
Please help to review this PR when you have time.

**Comment by chenrujun** [comment]
Hi, @backwind1233 , @saragluna , @yiliuTo , please help to review this PR when you have time.

**Review by saragluna — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/spring-security-support.adoc [hunk 1]
```diff
@@ -556,144 +658,53 @@ public class AadWebApplicationAndResourceServerConfig {
...
-Please refer to link:https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_{project-version}[azure-spring-boot-samples] for more details.
+This property(`spring.cloud.azure.active-directory.application-type`) is optional, its value can be inferred by dependencies, only `web_application_and_resource_server` must be configured manually: `spring.cloud.azure.active-directory.application-type=web_application_and_resource_server`.
 
+.Application type of spring-cloud-azure-starter-active-directory
+[cols="4*", options="header"]
+|===
+|Has dependency: spring-security-oauth2-client |Has dependency: spring-security-oauth2-resource-server |Valid values of application type |Default value
+|Yes |No |`web_application` |`web_application`
+|No |Yes |`resource_server` |`resource_server`
+|Yes |Yes |`web_application`,`resource_server`,`resource_server_with_obo`, `web_application_and_resource_server` |`resource_server_with_obo`
```
[the remark this answers is not in the document]
Comment (chenrujun) [reply docs/src/main/asciidoc/spring-security-support.adoc:705]: I didn't find way to achieve that.

Adding a ` ` after `,` will make the table display much better.

Before:
image

After:
image

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**